# Author: Ehsan Moravveji (VSC - KU Leuven)
# Purpose: The formatting of the 'license' token needs to be updated

diff -ruN dvc_ssh-4.2.2-orig/pyproject.toml dvc_ssh-4.2.2/pyproject.toml
--- dvc_ssh-4.2.2-orig/pyproject.toml	2025-10-24 15:12:06.480869000 +0200
+++ dvc_ssh-4.2.2/pyproject.toml	2025-10-24 15:12:28.716286000 +0200
@@ -13,7 +13,7 @@
     "dvc",
     "ssh",
 ]
-license = "Apache-2.0"
+license = { text = "Apache-2.0" }
 maintainers = [{ name = "Iterative", email = "support@dvc.org" }]
 authors = [{ name = "Iterative", email = "support@dvc.org" }]
 requires-python = ">=3.9"
